plugins/postcss-progressive-custom-properties/test/basic.expect.css

```diff
@@ -225,3 +225,28 @@
 	--color-functional-notation-percentage-alpha: rgba(0, 0, 0, 1);
 	--color-functional-notation-percentage-alpha: rgba(0, 0, 0, 100%);
 }
+
+:root {
+	/* color-mix() */
+	--color-mix-1: plum;
+	--color-mix-2: plum;
+}
+
+@supports (color: color(display-p3 0 0 0)) {
+:root {
+	--color-mix-1: color(display-p3 0.64331 0.19245 0.16771);
+}
+}
+
+@supports (color: color-mix(in lch, red, blue)) {
+:root {
+	--color-mix-1: color-mix(in lch, purple 50%, plum 50%);
+	--color-mix-2: color-mix(in lch longer, purple 50%, plum 50%);
+}
+}
+
+@supports (color: color(display-p3 0 0 0)) {
+:root {
+	--color-mix-2: color(display-p3 0.64331 0.19245 0.16771);
+}
+}
```

Comment:
With `display-p3` fallbacks a single declaration can produce 2 fallback declarations.

Combined with `progressive-custom-properties` it will produce alternating `@supports` rules.

I played around with an updated algorithm but it altered the overal order of CSS.

This test ensures I don't try the same thing again later without thinking about potential issues.